Lib/uuid.py

```diff
@@ -403,9 +403,16 @@ def _get_command_stdout(command, *args):
 #
 # See https://en.wikipedia.org/wiki/MAC_address#Universal_vs._local
 
+
 def _is_universal(mac):
     return not (mac & (1 << 41))
 
+# MAC adresses that are blacklisted for finding a node id
+_MAC_BLACKLIST={
+    # (GH-85724) Hardcoded MAC adres for all Intel MacBook's with Touch Bar.
```

Comment:
```suggestion
    # (GH-85724) Hardcoded MAC address for all Intel MacBook's with Touch Bar.
```

Reply:
Sigh... If I got a euro for every time I mistype "address" I'd be rich. Guess how one spells "address" in Dutch :-)